code/pizzabot.js

```diff
@@ -4,4 +4,88 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+var x = vegetarian
+var y = hawaiian
+var z = pepperoni
+
+alert("Hey! Happy to serve you pizza! On our menu we have " + x + ", " + y + " and " + z + ".");
+
+
+let orderName = prompt("Enter the name of the pizza you want to order today.");
+
+
+if (orderName.toLowerCase() == "vegetarian") {
+  modifiedName = x;
+} else if (orderName.toLowerCase() == "vegetarian pizza") {
+  modifiedName = x;
+} else if (orderName.toLowerCase() == "hawaiian") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "hawaiian pizza") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "hawaian") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "hawaian pizza") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "pepperoni") {
+  modifiedName = z;
+} else if (orderName.toLowerCase() == "pepperoni pizza") {
+  modifiedName = z;
+} else if (orderName.toLowerCase() == "peperoni") {
+  modifiedName = z;
+} else if (orderName.toLowerCase() == "peperoni pizza") {
+  modifiedName = z;
+} else {
+  alert('Select a pizza from the menu');
+  let orderName = prompt("Enter the name of the pizza you want to order today.");
+  if (orderName.toLowerCase() == "vegetarian") {
+    modifiedName = x;
+  } else if (orderName.toLowerCase() == "vegetarian pizza") {
+    modifiedName = x;
+  } else if (orderName.toLowerCase() == "hawaiian") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "hawaiian pizza") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "hawaian") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "hawaian pizza") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "pepperoni") {
+    modifiedName = z;
+  } else if (orderName.toLowerCase() == "pepperoni pizza") {
+    modifiedName = z;
+  } else if (orderName.toLowerCase() == "peperoni") {
+    modifiedName = z;
+  } else if (orderName.toLowerCase() == "peperoni pizza") {
+    modifiedName = z;
+  } else {
+    alert('Select a pizza from the menu');
+  }
+}
+
+
+const orderQuantity = prompt("How many of " + modifiedName + " do you want?");
+
+const totalCost = orderQuantity * 80;
+
+if (orderQuantity < 3) {
+  cookingTime = "10";
```

Comment:
When writing numbers you don't need `"` around. With `"` around it means that it's a string (text)
```suggestion
  cookingTime = 10;
```